.github/workflows/odh-konflux-onboarder.yml

```
@@ -81,6 +81,7 @@ on:
           - trainer-operator
           - trustyai-service
           - rhoai-mcp
+          - ''
```

actionlint (1.7.12)

[error] string should not be empty

(syntax-check)